js/templates/modals/purchase/purchase.html

```diff
@@ -19,6 +19,8 @@
     <div class="flexColRow gutterVMd">
       <% // to support multiple items in a purchase, loop this section %>
       <section class="contentBox pad clrP clrBr clrSh3">
+        <div class="js-errors"></div>
```

Comment:
This is the generic holder for errors that aren't associated with a specific section of the template.